src/omv/sensor.c

```diff
@@ -888,10 +892,57 @@ static void sensor_check_buffsize()
     }
 }
 
+// ARM Cortex-M4/M7 Processors can access memory using unaligned 32-bit reads/writes.
```

Comment:
It does, but it's actually slower.

Reply:
Yes, I found an algorithm that does 4 byte aligned writes and does barrel shifting to align things also. Maybe I'll have Larry implement that faster. This was less code and I didn't want to bring in that giant algorithm.